services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -36,6 +29,7 @@ class MessageWriterTest extends AbstractSdtUnitTestBase {
 
     private static final String UNIT_TEST = "UNITTEST";
 
+    private static final String DLQ_SUFFIX = "/$deadletterqueue";
     /**
      * MessageWriter for mocking.
      */
```

Comment:
Remove this comment - the MessageWriter class is not being mocked, so it isn't correct.